cmd/autofix.go

```diff
@@ -21,32 +21,51 @@ func getAuditFunctions() []interface{} {
 // preserves the order of the keys) using the Shopify/yaml fork of go-yaml/yaml (the fork adds comment support) and
 // then merge the fixed MapSlice back into the original MapSlice so that we get the comments and original order back.
 func autofix(*cobra.Command, []string) {
+
+	var toAppend bool
```

Comment:
I didn't explicitly set it to False since nil value for Bool is false and I don't think we were setting toAppend to false in the earlier version, but I guess it wouldn't hurt to explicitly set it to false :)

Reply:
This is purely opinion based and I have definitely heard arguments for both sides. Personally, I think explicitly setting values even for the default improves readability but feel free to leave it as is if you like